### ISSUE TITLE
Add Mac OS as a platform

### ISSUE THREAD

**Issue by jcampbell05** [react-everywhere/re-start#46]
You have the UMP platform supported but there is also a great library for adding native MacOS support that would be awesome to be included.

https://github.com/ptmt/react-native-macos

**Comment by piranna** [comment]
Pull-requests are welcome :-D I think it would be cool if you add the support on the [re-base](https://github.com/react-everywhere/re-start/tree/react-16/templates/re-base) template :-)

**Comment by jcampbell05** [comment]
Sure :) would love to :) is there a document I should read before I begin ?

**Comment by piranna** [comment]
> Sure :) would love to :) is there a document I should read before I begin ?

I don't think so, just only add the dependencies that you need on that template and use any of its `index.js` files as basis to make it work on OSX :-)
